test/integration/sequencer_test.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require("test_helper")
+# require("capybara_helper")
+
+# Test adding, editing, and deleting a Sequence
+# Uses Capybara; MO SessionFormExtensions don't work on form_with forms
+# Use an integration test because click_mo_link needs an id, and
+# there's no  id (at least by default) for a form_with resource-oriented
+# form field.
```

Comment:
@pellaea:
### Should I try to modify SessionFormExtensions so that `click_mo_link` etc.  work on `form_with` forms?
(That probably belongs in another PR.)

Reply:
Shouldn't we be migrating away from our custom-rolled integration helpers to Capybara and (now) normal integration tests anyway?  I think I updated click_mo_link a few weeks/months ago because it was the quickest way to get some big PR of Nimmo's working.  Not because I want to be maintaining out-dated test helpers!

Reply:
I agree.
(Even though I think there are downsides:
- The home-rolled helpers feel faster than Capybara.
- The home-rolled helpers sometimes do some extra testing.)